### DISCUSSION TITLE
Set cc and bcc

### DISCUSSION THREAD

**Discussion by JannyM85** [Jericho/StrongGrid#373] [Q&A] [answered]
Hello, it's possible to send email with cc and bcc? In sendgrid API it is possible but, i don't know how to do by your code. Thanks.

**Comment by JannyM85** [comment, marked as answer]
On the end i found SendAsync method :)

**Reply by Jericho** [reply]
Exactly what I was going to suggest!

The other methods like `SendToSingleRecipientAsync`, `SendToMultipleRecipientsAsync`, etc. are for convenience. I have included parameters that are most commonly used (in my opinion).

Also, I hope you noticed the following remark in the XML documentation for each of these 'convenience' methods:
> /// <remarks>
/// This is a convenience method with simplified parameters.
/// If you need more options, use the "SendAsync" method.
/// </remarks>

Glad you figured it out. Cheers!
